backend/molgenis-emx2-semantics/src/test/java/org/molgenis/emx2/semantics/fairdatapoint/FAIRDataPointBadDistributionInDatasetTest.java

```diff
@@ -69,8 +69,9 @@ public void FDPBadDistribution() throws Exception {
     Exception exception =
         assertThrows(
             Exception.class,
-            () -> new FAIRDataPointDataset(
-                request, fairDataHub_baddistribution.getTable("FDP_Dataset")));
+            () ->
+                new FAIRDataPointDataset(
+                    request, fairDataHub_baddistribution.getTable("FDP_Dataset")));
```

Comment:
other PR

Reply:
This kind of stuff keeps happening, I'm getting really tired of keeping track of it, so I'd rather leave it in.

Reply:
i do not agree, i made a note of formatting on the slack and this was fixed, if we need to change stuff in the CI setup or make changes to the work flow we need to do this , but this seems like a rebase issue